plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/FutureNonSystemJobSuspenderTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.appengine.facets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.core.runtime.jobs.Job;
+import org.junit.After;
+import org.junit.Test;
+
+public class FutureNonSystemJobSuspenderTest {
+
+  private Job job1 = new NoOpSpinJob("Test job 1");
+  private Job job2 = new NoOpSpinJob("Test job 2");
+
+  @After
+  public void tearDown() {
+    FutureNonSystemJobSuspender.resumeInternal();
+    job1.cancel();
+    job2.cancel();
+  }
+
+  @Test
+  public void testCannotSuspendConcurrently() {
+    FutureNonSystemJobSuspender.suspendFutureJobs();
+    try {
+      FutureNonSystemJobSuspender.suspendFutureJobs();
+      fail();
+    } catch (IllegalArgumentException expected) {}
+  }
+
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
Why do we use the expected annotation here and not in the previous similar test case?